components/contentComponents/playgroundSearchResult.tsx

```diff
@@ -103,32 +103,35 @@ export const PlaygroundSearchResult: FunctionComponent<{ searchResult: IPlaygrou
 
     useEffect(() => {
         // get the snippet code
-        fetch(`https://snippet.babylonjs.com/${searchResult.snippetIdentifier}/${searchResult.version}`).then((response) => {
-            response.text().then((text) => {
-                const parsed = JSON.parse(JSON.parse(text).jsonPayload);
-                const { code } = parsed;
-                const codeLines: Array<string> = (code || "").split("\n");
-                let startingLine = 0;
-                let foundLine = -1;
-                const lowerTerm = term.toLowerCase();
-                if (type === "code") {
-                    for (foundLine = 0; foundLine < codeLines.length; ++foundLine) {
-                        if (codeLines[foundLine].length > 100) {
-                            codeLines[foundLine] = `${codeLines[foundLine].substring(0, 100)}...`;
-                        }
-                        if (codeLines[foundLine].toLowerCase().indexOf(lowerTerm) !== -1) {
-                            startingLine = Math.max(foundLine - 5, 0);
-                            break;
-                        }
+        fetch(`https://snippet.babylonjs.com/${searchResult.snippetIdentifier}/${searchResult.version}`)
+        .then(response => response.text())
+        .then(text => {
+            const parsed = JSON.parse(JSON.parse(text).jsonPayload);

```

Comment:
would this work?

```suggestion
        .then(response => response.json())
        .then(jsonData => {
            const parsed = JSON.parse(jsonData.jsonPayload);
```

Reply:
Both should work, one needs one less JSON.parse call.